config/default.schema.json

```diff
@@ -279,6 +279,11 @@
 			"default": "",
 			"description": "A string with concatenated user types (student, employee) that don't need to explicitly state their consent with the terms and conditions. \"employee\" means teachers and admins"
 		},
+		"SESSION_COOKIE_SAME_SITE": {
+			"type": "string",
+			"default": "strict",
+			"description": "express-session cookie samesite attribute"
+		},
```

Comment:
I would put this in the COOKIE group just below

Reply:
you can not apply the same logic to all cookies and some need more protection than others